source/clear-linux/get-started/get-started.rst

```diff
@@ -0,0 +1,23 @@
+.. _get-started:
+
+Get started
+###########
+
+This section is where you will find documentation to do with the
+installation of |CL|.
+
+The :ref:`get-started` section provides end to end instructions for
+downloading and running |CL| on :ref:`bare metal <bare-metal-install>`, under
+a :ref:`virtual machine <virtual-machine-install>`, or by way of :ref:`live image <live-image>`.
+It is also home to single task instructions and other useful information
```

Comment:
Consider changing to: Additionally, it provides useful pre-install information and instructions on how to complete pre-install tasks.